README.md

```diff
@@ -39,23 +39,26 @@ Windows users need a Python installation and C++ compiler:
     ```bash
     cd spdx-online-tools
     pip install -r requirements.txt
-    ```
-4. Run Django migrate.
+
+4. Create Github API configuration according to [GitHub Developer Sensitive Data](#github-developer-sensitive-data).
```

Comment:
Please include the correct link

Reply:
I'm actually linking to a section that is already available in the readme. This format works in my editor and should also work on GitHub: https://til.hashrocket.com/posts/3d5f989178-link-to-headers-in-github-readmes

Would you the full link, such as https://github.com/spdx/spdx-online-tools#github-developer-sensitive-data ?